R/load_experiment.R

```diff
@@ -89,11 +85,11 @@ load_experiment <- function(exp, model, dir = "") {
     names(x) <- dic_g2r
     list(out = x, dic_g2r = dic_g2r)
   }
-  variables <- make_df_dic(get_variables(out)[get_variables_names(model)])
-  parameters <- make_df_dic(get_parameters(out)[get_parameters_names(model)])
+  variables <- make_df_dic(get_variables(out))
```

Comment:
I thought it was necessary because it was discriminating the observed variables and the display. But, after discussion with Nicolas, we know that the display has the same behavior as observed variables in `rama` so we don't need to discriminate them so the function `get_names` are not used anymore.